metriq_gym/origin/device.py

```diff
@@ -15,25 +15,140 @@
 from .job import OriginJob
 from .qcloud_utils import get_qcloud_options
 
+if TYPE_CHECKING:  # pragma: no cover
+    from pyqpanda3.qcloud import QCloudBackend
```

Comment:
no need for this to be under TYPE_CHECKING, since we already import pyqpanda3 in L7